app/signals.js

```diff
@@ -42,8 +42,8 @@ export default function registerSignals(controller) {
     //tnr: NOT YET WORKING:
     //higher priority
     controller.signal('pasteSequenceString', [a.pasteSequenceString]);
-    controller.signal('setSelectionLayer', [a.setSelectionLayer]);
-    controller.signal('toggleAnnotationDisplay', [a.setCaretPosition]);
+    controller.signal('setOrClearSelectionLayer', [a.setSelectionLayer]);
```

Comment:
I think this signal should still be `setSelectionLayer` and not `setOrClearSelectionLayer`